mobile/lib/modules/home/views/home_page.dart

```diff
@@ -76,9 +78,27 @@ class HomePage extends HookConsumerWidget {
         selectionEnabledHook.value = false;
       }
 
+      Iterable<Asset> remoteOnlySelection() {
```

Comment:
because `return assets.where((a) => a.isRemote);` is neither a Set nor a List. Its an on-the-fly computed "stream" of assets -> avoids memory allocations & reduces memory usage -> performance optimization.

Unless the user selects hundreds or thousands of assets, you probably won't notice. But: it is always faster... unless you iterate the "stream" multiple times afterwards.